ui/next.config.js

```diff
@@ -9,6 +9,9 @@ module.exports = phase => ({
 
     return null
   },
+  async redirects() {
+    return [{ source: '/', destination: '/destinations', permanent: true }]
```

Comment:
LGTM overall – will this work as well in scenarios where the user is logged out?

Reply:
Oh, it is a good catch! I added the conditional check in the next.config.js - however this might not work for the social login yet due to this bug - https://github.com/infrahq/infra/issues/4003